Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4168,3 +4170,21 @@ bool CLuaVehicleDefs::BlowVehicle(CClientEntity* entity, std::optional<bool> wit
 {
     return CStaticFunctionDefinitions::BlowVehicle(*entity, withExplosion);
 }
+
+std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::GetVehicleEntryPoints(CClientVehicle* vehicle)
+{
+    if (CClientVehicleManager::GetMaxPassengerCount(vehicle->GetModel()) == 255)
+    {
+        return false;
+    }
+
+    std::array<CVector, 4> entryPoints;
+
+    std::uint32_t i = 0;
+    for (auto& entryPoint : entryPoints)
+    {
+        entryPoint = vehicle->GetEntryPoint(i++);
+    }
+
+    return entryPoints;
```

Comment:
You said it doesnt return an array? If `std::array` is not supported then you could use `std::vector` instead

Reply:
I more so meant that if I return CVector here as part of the array then that's also exactly what I'm getting in MTA. However, that's not intended to be the case for a non-OOP function. That's why I used to have a separate function for the OOP version. Note that this was written years ago when things were still a bit different.

Reply:
> I more so meant that if I return CVector here as part of the array then that's also exactly what I'm getting in MTA. However, that's not intended to be the case for a non-OOP function. That's why I used to have a separate function for the OOP version. Note that this was written years ago when things were still a bit different.

so what do you expect the POP function to return?